src/neuroconv/tools/nwb_helpers.py

```diff
@@ -47,20 +50,35 @@ def get_default_nwbfile_metadata() -> DeepDict:
 
 def make_nwbfile_from_metadata(metadata: dict) -> NWBFile:
     """Make NWBFile from available metadata."""
-    metadata = dict_deep_update(get_default_nwbfile_metadata(), metadata)
+
+    # Validate metadata
+    schema_path = Path(__file__).resolve().parent.parent / "schemas/base_metadata_schema.json"
+    with open(schema_path) as schema_file:
+        base_metadata_schema = json.load(schema_file)
+    try:
+        # jsonschema.validate(metadata, base_metadata_schema)
+        validate_metadata(metadata=metadata, schema=base_metadata_schema)
+    except jsonschema.exceptions.ValidationError as e:
+        print(f"Base metadata is not valid.\n")
+        raise e
```

Comment:
Point 1: other people on the NeuroConv team feel strongly against default verbosity (which is now `False` BTW); need to either add control of that `print` statement or, probably better, just remove it and let the error do the talking

Point 2: Why have a try/except here at all? What is it avoiding? Especially since a non-ValidationError would get raised as normal and the only different is the print statement which should probably just be removed as per Point 1

Reply:
fair point, I removed it